site/docs/components/toast/examples.mdx

```diff
@@ -49,6 +49,13 @@ Use the success toast to communicate the successful completion of an action. A s
 
 </LivePreview>
 
+<LivePreview componentName="toast" exampleName="CustomIcon" >
+
+## Custom Icon
+
+Use the `Icon` prop to customize the icon shown in the toast, while still respecting the status prop.
```

Comment:
@mark-tate, how does this sound?
```suggestion
Use the `Icon` prop to customize the icon shown in the toast, while leaving the `status` prop unchanged.
```

Reply:
what does it mean it leaves the status unchanged? that it will keep the styling of the status and the icon will inherit the colour?

Reply:
it does, in the original change styles were being passed to the custom icon, but i think we might want to remove this statement all together, since now the status is just being inherited

Reply:
Something like:
Use the `icon` prop to customize the icon shown in the toast. If you also provide a `status` prop....

What happens if you don't have status btw?

Reply:
> Something like: Use the `icon` prop to customize the icon shown in the toast. If you also provide a `status` prop....
> 
> What happens if you don't have status btw?

no status toasts don't have icons